tests/integration/bond_test.py

```diff
@@ -404,6 +408,26 @@ def test_replacing_port_keeps_mac_of_existing_bond(bond99_with_eth2, eth1_up):
     )
 
 
+@pytest.mark.skipif(
+    nm_major_minor_version() < 1.30,
+    reason="On NM lower than 1.30, the bond keeps the existing mac",
+)
+def test_replacing_port_modify_mac_of_existing_bond(bond99_with_eth2, eth1_up):
+    desired_state = bond99_with_eth2
+    bond_state = desired_state[Interface.KEY][0]
+    eth1_name = eth1_up[Interface.KEY][0][Interface.NAME]
+    bond_state[Bond.CONFIG_SUBTREE][Bond.PORT] = [eth1_name]
+
+    current_state = statelib.show_only((bond_state[Interface.NAME],))
+
+    libnmstate.apply(desired_state)
+    modified_state = statelib.show_only((bond_state[Interface.NAME],))
+    assert (
+        modified_state[Interface.KEY][0][Interface.MAC]
+        != current_state[Interface.KEY][0][Interface.MAC]
```

Comment:
This should probably assert which mac we are expecting instead of only asserting that it will change.

Reply:
I am dropping both tests. As we do not care anymore about this specific case.